user-profile-card.js

```diff
@@ -120,23 +122,28 @@ class UserProfileCard extends LocalizeUserProfileCard(RtlMixin(LitElement)) {
 		const openAlert = !this._isOpen ? this.localize('profileCardClosed') : this.localize('profileCardOpened');
 
 		return html`
-		<d2l-dropdown open-on-hover class="d2l-labs-user-profile-card-dropdown">
-			<d2l-profile-image size="${ifDefined(this._getSize())}"
-			role="button"
-			aria-expanded="${this._isOpen}"
-			aria-haspopup="true"
-			aria-label="${this.localize('profileCardOpener', { displayName : this._displayName })}"
-			class="d2l-labs-user-profile-card-opener d2l-dropdown-opener"
-			href=${this._userHref}
-			.token=${this.token}
-			tabindex="0"
-			@keydown="${this._onOpenerKeypress}"
-			@click="${this._onOpenerClicked}"
-			></d2l-profile-image>
+		<d2l-dropdown
+			slotted-opener
+			open-on-hover="${ifDefined(this.noOpenOnHover ? undefined : true)}"
+			class="d2l-labs-user-profile-card-dropdown">
+				<slot name="opener" @slotchange="${this._onOpenerSlotChange}">
+					<d2l-profile-image size="${ifDefined(this._getSize())}"
+						role="button"
+						aria-expanded="${this._isOpen}"
+						aria-haspopup="true"
+						aria-label="${this.localize('profileCardOpener', { displayName : this._displayName })}"
+						class="d2l-labs-user-profile-card-opener d2l-dropdown-opener"
+						href=${this._userHref}
+						.token=${this.token}
+						tabindex="0"
+						@keydown="${this._onOpenerKeypress}"
+						@click="${this._onOpenerClicked}"
+					></d2l-profile-image>
+				</slot>
 				<d2l-dropdown-content
 					mobile-tray="bottom"
 					?trap-focus="${this._trapFocus}"
-					no-auto-close
+					no-auto-close="${ifDefined(this.noOpenOnHover ? undefined : true)}"
```

Comment:
```suggestion
					?no-auto-close="${!this.noOpenOnHover}"
```
Same here, the `?` should tell Lit to handle whether to render the boolean attribute or not